scripts/check_maintainers.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2024 Vestas Wind Systems A/S
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import sys
+
+from github_helpers import get_github_object
+from get_maintainer import Maintainers
+from github.GithubException import UnknownObjectException
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        description=__doc__, allow_abbrev=False)
+
+    parser.add_argument(
+        "-m", "--maintainers",
+        metavar="MAINTAINERS_FILE",
+        help="Maintainers file to load. If not specified, MAINTAINERS.yml in "
+             "the top-level repository directory is used, and must exist. "
+             "Paths in the maintainers file will always be taken as relative "
+             "to the top-level directory.")
+
+    return parser.parse_args()
+
+def main() -> None:
+    args = parse_args()
+    zephyr_repo = get_github_object().get_repo('zephyrproject-rtos/zephyr')
+    maintainers = Maintainers(args.maintainers)
+    gh = get_github_object()
+    gh_users = []
+    notfound = []
+    noncollabs = []
+
+    for area in maintainers.areas.values():
+        gh_users = list(set(gh_users + area.maintainers + area.collaborators))
+
+    gh_users.sort()
+
+    print('Checking maintainer and collaborator user accounts on GitHub:')
+    for gh_user in gh_users:
+        try:
+            print('.', end='', flush=True)
+            gh.get_user(gh_user)
+
+            if not zephyr_repo.has_in_collaborators(gh_user):
```

Comment:
Is this right? Collaborators and org members are not the same, and maintainers for example are org members but not collaborators?

Reply:
I guess it will work since `affiliation` is `all` by default:
https://docs.github.com/en/rest/collaborators/collaborators?apiVersion=2022-11-28#list-repository-collaborators

Reply:
Correct.